cw/capture.py

```diff
@@ -258,6 +274,7 @@ def capture_aes_random_batch(ot, ktp, capture_cfg, scope_type):
     num_segments_storage = 1
     # cw and waverunner scopes are supported fot batch capture.
     scope = SCOPE_FACTORY[scope_type](ot, capture_cfg)
+
```

Comment:
I think this additioan newline (and the other two below) aren't required. I don't mind them being there to improve readability but then they should be added with the next commit which touches the surroundings of these lines anyway.

Reply:
I see, but I don't understand which next commit you mean.

Reply:
Ah sorry, got it!